src/libs/sfdk/virtualmachine.cpp

```diff
@@ -677,6 +675,18 @@ VirtualMachinePrivate::~VirtualMachinePrivate()
 {
 }
 
+void VirtualMachinePrivate::initGuest(const QObject *context, const Functor<> &functor)
+{
+    BatchComposer composer = BatchComposer::createBatch("VirtualMachine::initGuest");
+    const QPointer<const QObject> context_{context};
+
+    // Errors not considered fatal here
+    doInitGuest();
+    emit initGuest();
```

Comment:
I'm quite sure this signal is no longer connected anywhere, and could be just removed

Reply:
Yup I am aware of that - preserving it as a guidance for possible future use.